apps/odyssey-stats/src/widget/index.scss

```diff
@@ -1,7 +1,3 @@
 @import "../styles/theme.scss";
```

Comment:
I see the same problems reported by @a8ck3n; we'll want to limit our styles to our widget and avoid affecting the rest of the dashboard.

Reply:
Created [an issue](https://github.com/Automattic/wp-calypso/issues/75360) to track it and we'll address in a follow up PR.